app/pages/notifications.spec.js

```diff
@@ -30,18 +32,20 @@ describe('Notifications', function() {
     });
 
     it('will notify when no notifications present', function() {
+      const stub = sinon.stub(Notifications.prototype, 'componentWillMount');
       wrapper = mount(<Notifications user={{ id: 1 }} />);
       assert(wrapper.contains(<span>You have no notifications.</span>));
+      stub.restore();
     });
   });
 
   describe('it correctly display projects', function() {
     beforeEach(function () {
       wrapper = shallow(
-        <Notifications user={{ id: 1 }} />,
```

Comment:
One of the options for `shallow` is to disable lifecycle methods too: https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md

Which I think you might want in this case since you want to pass the mocked response and not have that API call run.